modules/tensor_mechanics/src/materials/ComputeIsotropicElasticityTensor.C

```diff
@@ -0,0 +1,65 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "ComputeIsotropicElasticityTensor.h"
+
+template<>
+InputParameters validParams<ComputeIsotropicElasticityTensor>()
+{
+  InputParameters params = validParams<ComputeElasticityTensorBase>();
+  params.addClassDescription("Compute an isotropic elasticity tensor.");
+  params.addParam<Real>("bulk_modulus", "The bulk modulus for the material.");
+  params.addParam<Real>("lambda", "Lame's first constant for the material.");
+  params.addParam<Real>("poissons_ratio", "Poisson's ratio for the material.");
+  params.addParam<Real>("shear_modulus", "The shear modulus of the material.");
+  params.addParam<Real>("youngs_modulus", "Young's modulus of the material.");
+  return params;
+}
+
+ComputeIsotropicElasticityTensor::ComputeIsotropicElasticityTensor(const std::string & name,
+                                                 InputParameters parameters) :
+    ComputeElasticityTensorBase(name, parameters),
+  _bulk_modulus_set( parameters.isParamValid("bulk_modulus") ),
+  _lambda_set( parameters.isParamValid("lambda") ),
+  _poissons_ratio_set( parameters.isParamValid("poissons_ratio") ),
+  _shear_modulus_set( parameters.isParamValid("shear_modulus") ),
+  _youngs_modulus_set( parameters.isParamValid("youngs_modulus") ),
+  _bulk_modulus( _bulk_modulus_set ? getParam<Real>("bulk_modulus") : -1 ),
+  _lambda( _lambda_set ? getParam<Real>("lambda") : -1 ),
+  _poissons_ratio( _poissons_ratio_set ?  getParam<Real>("poissons_ratio") : -1 ),
+  _shear_modulus( _shear_modulus_set ? getParam<Real>("shear_modulus") : -1 ),
+  _youngs_modulus( _youngs_modulus_set ? getParam<Real>("youngs_modulus") : -1 )
+{
```

Comment:
I know this is a nitpick, but your spacing is wrong in the initialization list here. Should be four spaces.